opendatahub-io/odh-dashboard
# Bring popovers to kserve modal

## Description



## How Has This Been Tested?



Closes #2091 

## Test Impact


1. Deploy a new kserve model
2. Check that the popovers present in modelmesh are in this modal too


<img width="823" alt="Screenshot 2023-12-01 at 14 48 21" src="https://github.com/opendatahub-io/odh-dashboard/assets/16117276/d9ccfa2e-f455-4d59-a04b-8b0578f5adcd">


## Request review criteria:



Self checklist (all need to be checked):

If you have UI changes: 


After the PR is posted & before it merges:
